index.html

```diff
@@ -5,126 +5,111 @@
+    </header>
+
     <!-- main article section -->
 
-    <section id="main-info">
-      <p class="main-article">
-        <br />
-        <b>
-          Even if i knew that tomorrow the world go to pieces , I would like
-          still plant my apple . Martin Luther King
-        </b>
-        <br />
-        <br />
-        Trees are one of the most beautiful plants on our planet. Since the
-        beginning, trees have furnished us with two of life’s essentials, food
-        and oxygen. As we evolved, they provided additional necessities such as
-        shelter, medicine, and tools. Today, their value continues to increase
-        and more benefits of trees are being discovered as their role expands to
-        satisfy the needs created by our modern lifestyles.Can you imagine how
-        many trees surround us, how many benefits they bring to us.A study in
-        the journal of ‘Nature’ reported close to 3.04 Trillion trees on earth.
-        And though this might seem like a lot- it’s not! 3.04 Trillion trees
-        make for almost 422 trees per person.The largest forest in the world
-        exists far to the north amongst a less bountiful ecosystem. The Great
-        Boreal Forests span the Arctic circle in Canada, Russia, and
-        Scandinavia. These forests contain approximately 750 billion trees,
-        nearly ¼ of the world’s total.Forests are very important to the
-        environment because they use carbon dioxide (CO2) and produce oxygen.
-        They also provide shelter and food for many different types of plants
-        and animals.
-        <br />
-        <br />
-        But we also have negative aspects like cutting down trees is called
-        deforestation and it can have serious effects. If animals’ homes, which
-        are built in and around the trees, are destroyed, many species will
-        become extinct. For example, many different types of animals in South
-        America were lost before scientists could learn about them.
-        Deforestation also affects the climate. When rain falls on a forest, new
-        clouds are created. If huge areas of trees are cut down, clouds do not
-        form, rain does not fall and the land becomes drier. The CO2 builds up
-        and adds to the global warming which is making the Earth become hotter.
-        <br />
-        <br />
-        Many European countries are planting trees faster than they are cutting
-        them down. Many trees were planted a few years ago to replace woodland
-        and to create forests in areas where there wasn’t much wood. Have you
-        thought about what we can do to make our forests and parks even more
-        beautiful and more save ? If not, please use our site to improve your
-        knowledge of trees and understand what benefits we can do to improve the
-        environment.
-        <br />
-        <br />
-      </p>
-    </section>
+    <section>
```

Comment:
instead of using  "`<section>  <div class="info">`", you can use [main tag]( https://www.w3schools.com/tags/tag_main.asp)

Reply:
Main, Section, Div, Article this are all HTML tag. their functions are the same. No problem